src/stores/AutoRageshakeStore.ts

```diff
@@ -99,7 +99,7 @@ export default class AutoRageshakeStore extends AsyncStoreWithClient<IState> {
                 userText: "Auto-reporting decryption error (recipient)",
                 sendLogs: true,
                 labels: ["Z-UISI", "web", "uisi-recipient"],
-                customApp: "element-auto-uisi",
+                customApp: SdkConfig.get().uisi_autorageshake_app || "element-web",
```

Comment:
Might be saner if `sendBugReport` does `customApp || "element-web"` instead to prevent duplication/accidentally missing a customApp fallback elsewhere